src/switch/README.md

```diff
@@ -0,0 +1,447 @@
+libp2p-switch JavaScript implementation
+======================================
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![Discourse posts](https://img.shields.io/discourse/https/discuss.libp2p.io/posts.svg)](https://discuss.libp2p.io)
+[![Travis CI](https://flat.badgen.net/travis/libp2p/js-libp2p-switch)](https://travis-ci.com/libp2p/js-libp2p-switch)
```

Comment:
Should we remove CI, cdecov and dependency status?

Reply:
Yes! I will go through and sweep the READMEs and cleanup the package.json files as part of the next PR to pull everything else in. If we keep the package.json files it may just be a super stripped down version, as the main reason I left them in is to be able to keep the clean `libp2p-switch` require statements. But that might be unnecessary once everything is cleaned up.